app/src/main/java/org/oppia/app/help/HelpActivityPresenter.kt

```diff
@@ -17,10 +17,12 @@ class HelpActivityPresenter @Inject constructor(private val activity: AppCompatA
   fun handleOnCreate() {
     activity.setContentView(R.layout.help_activity)
     setUpNavigationDrawer()
-    activity.supportFragmentManager.beginTransaction().add(
-      R.id.help_fragment_placeholder,
-      HelpFragment()
-    ).commitNow()
+    if (getHelpFragment() == null) {
```

Comment:
Changes in this file:
Prefer not using null, I missed this in last PR review.
`private lateinit var navigationDrawerFragment: NavigationDrawerFragment`

Reply:
This will bring change in `setUpNavigationDrawer` also
`navigationDrawerFragment.setUpDrawer(`

Reply:
Done